examples/worker_rush.py

```diff
@@ -1,12 +1,18 @@
 import sc2
-from sc2 import run_game, maps, Race, Difficulty
+from sc2 import run_game, maps, Race, Difficulty, BotAI
 from sc2.player import Bot, Computer
 
-class WorkerRushBot(sc2.BotAI):
+class WorkerRushBot(BotAI):
     async def on_step(self, iteration):
+        actions = []
```

Comment:
Please convert `actions` to a class variable `self.actions` and initialize it in the `__init__` :)

Reply:
Sure. I actually did it like that but thought it would be too many changes. 😛